### ISSUE TITLE
Different Share Icon depending on loged in/of state

### ISSUE THREAD

**Issue by SerenaButler** [LycheeOrg/Lychee-Laravel#134]
![image](https://user-images.githubusercontent.com/35893306/54084493-9728c200-4331-11e9-87b6-8de6090c210b.png)

**Comment by ildyria** [comment]
I did not chose the icons, electrious did. I think the eye is a reference to "visibility".

**Comment by SerenaButler** [comment]
Yes but in includes the same sharing subset as when not logged. It's a bit inconsistent. Anyway :-)

**Comment by d7415** [comment]
My understanding is that the eye started as public/private, but then sharing options were added there. It makes no sense to be the eye when not logged in, as you cannot set that.

@bennettscience was suggesting we change it in LycheeOrg/Lychee-front#26. I quite like the eye. I'm generally against extra menus, but maybe it makes sense here to split owner-level and public-level "sharing" options.
